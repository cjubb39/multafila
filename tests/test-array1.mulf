/* array test */
int main(){
	
	int x[2];
	int y[2];
	
	x[0] = 1;
	x[1] = 2;
	x[2] = 3;
	y[0] = 4;
	y[1] = 5;
	y[2] = 6;
	
	int result = x[0] + y[1];
	printOut(result); /* should print 6 */
	
	return 1;
	
}