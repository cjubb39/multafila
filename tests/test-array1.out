
/*==========OUTPUT CODE BELOW==========*/
#include <stdio.h>
#include <pthread.h>
typedef int mf_bool;
#define mf_true 1; 
#define mf_false 0; 
void printOut(char *m){printf("%s\n", m);}
void printInt(int m){printf("%d\n", m);}

int main (  );

pthread_mutex_t global_lock;

pthread_cond_t global_cond;

struct global_lock_struct_s{
} global_lock_struct;

pthread_t global_threads [0];
int main (  )
{

{
pthread_mutex_init( & global_lock , NULL);
pthread_cond_init( & global_cond , NULL);
}
 ;
int   x[2] ;
int   y[2] ;
(null) = 1 ;
(null) = 2 ;
(null) = 3 ;
(null) = 4 ;
(null) = 5 ;
(null) = 6 ;
result = (null) + (null) ;
printOut( result ) ;
return 1 ;
}

